showcase/app/components/mock/components/form/super-select/generic-content.gts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import type { TemplateOnlyComponent } from '@ember/component/template-only';
```

Comment:
```suggestion
import type { TOC } from '@ember/component/template-only';
```

I know it's used else where with fully typed name but this is a little easier IMO

Reply:
TOC is an acronym (eg. could be Table Of Contents) so better to use the extended version, does what it says on the tin